### PR TITLE
Update partner page redirect links 

### PR DESCRIPTION
**Pull request by tejavar**
### Proposed changes

Change oracle redirect to Pulumi Page instead of generic page. 
Change Snyk's redirect to their Partner Page instead of redirect back to Pulumi 
Reorder logos alphabetically (keeping AWS, Azure and GCP on the top)

### CODE REVIEW

File: data/partners/partners.yaml [hunk 1]
```diff
@@ -3,89 +3,93 @@
...
-  logo: /logos/tech/newrelic.svg
+- name: Chronosphere
+  link: https://docs.chronosphere.io/administer/infrastructure/pulumi
+  logo: /logos/tech/chronosphere.svg
 
-- name: Snyk
-  link: https://docs.snyk.io/integrate-with-snyk/vulnerability-management-tools/pulumi-integration/
-  logo: /logos/tech/snyk.png
+- name: Civo
+  link: https://www.civo.com/
```
Comment (isaac-pulumi) [inline data/partners/partners.yaml:30]:
```suggestion
  link: https://www.civo.com/learn/automate-iac-pulumi-python-github-actions
```

File: data/partners/partners.yaml [hunk 2]
```diff
@@ -3,89 +3,93 @@
...
-  logo: /logos/pkg/github.svg
-
-- name: Snowflake
-  link: https://www.snowflake.com/
-  logo: /logos/pkg/snowflake.svg
-
-- name: Zscaler
-  link: https://www.zscaler.com/
-  logo: /logos/tech/zscaler.svg
+- name: Dagger
```
Comment (isaac-pulumi) [inline data/partners/partners.yaml:37]: Let's make sure they publish docs. Josh has working example

File: data/partners/partners.yaml [hunk 3]
```diff
@@ -3,89 +3,93 @@
...
-  logo: /logos/tech/chronosphere.svg
+- name: Pinecone
+  link: https://docs.pinecone.io/integrations/pulumi
+  logo: /logos/pkg/pinecone.svg
 
-- name: 1Password
-  link: https://1password.com/
-  logo: /logos/tech/1password.svg
+- name: Snowflake
+  link: https://www.snowflake.com/
```
Comment (isaac-pulumi) [inline data/partners/partners.yaml:74]: Investigate why we were removed from their partner page.

File: data/partners/partners.yaml [hunk 4]
```diff
@@ -3,89 +3,93 @@
...
-- name: Checkly
-  link: https://www.checklyhq.com/
-  logo: /logos/tech/checkly.svg
-
-- name: Civo
-  link: https://www.civo.com/
-  logo: /logos/pkg/civo2.svg
\ No newline at end of file
+- name: Zscaler
+  link: https://www.zscaler.com/partners/technology/cloud#devops
```
Comment (isaac-pulumi) [inline data/partners/partners.yaml:94]: Find out how to get a solutions brief link and other links similar to Hashi.

Comment (tejavar) [reply data/partners/partners.yaml:94]: Yes ! 
Our designer is working on this right now and this is something we will be adding to our partner page next quarter.

File: data/partners/partners.yaml (hunk 3 above)
Comment (tejavar) [reply data/partners/partners.yaml:74]: I don’t currently have a Snowflake contact but would be happy to engage if you can connect me with the right person. Let me know!

**Review by isaac-pulumi — APPROVED** [review]
Let's open separate issues to track:
1) Getting a dedicated landing page on each partner site (especially where they've committed to docs or where our past partner page has disappeared).

**Pull request merged**